pkg/operation/cloudbotanist/awsbotanist/infrastructure.go

```diff
@@ -149,8 +148,8 @@ func (b *AWSBotanist) generateTerraformBackupConfig() map[string]interface{} {
 			"region": b.Seed.Info.Spec.Cloud.Region,
 		},
 		"bucket": map[string]interface{}{
-			"name": fmt.Sprintf("%s-%s", b.Shoot.SeedNamespace, utils.ComputeSHA1Hex([]byte(b.Shoot.Info.Status.UID))[:5]),
+			"name": b.Operation.BackupInfrastructure.Name,
 		},
-		"clusterName": b.Shoot.SeedNamespace,
+		"clusterName": b.Operation.BackupInfrastructure.Name, //CHECKDIFF
```

Comment:
What will happen to the existing buckets? Will they be recreated? How does Terraform behave?